### ISSUE TITLE
Am I the only one not catching any PKMN at all?

### ISSUE THREAD

**Issue by MDashK** [SwipeX/PokeMate#320]
I'm pretty sure I'm not softbanned or anything like that, because I already checked the account on my real phone. But whenever I open the bot, I don't seem to catch any PKMN. And I'm running it in a very well crowded area...
Stops are tagged, eggs hatch, items are collected... But not a single PKMN is caught.. and I have loads of balls...
Walk speed is set to 0.50, it used to be 2.00, and still, at the time I would catch everything... But even at 0.50 it's not catching anything...

**Comment by SippieCup** [comment]
There is  maximum number of pokemon that can be caught within a certain time limit. Are you sure you haven't hit that limit?

**Comment by MDashK** [comment]
Nope.. Not at all.
Yesterday, I was in the said area personally with my phone. Catched a bunch of them. But nothing more than maybe 15 or so.
Today, early in the morning, opened in my house. Everything was OK. Didn't caught any PKMN.
Hours later, opened the bot in that location I was yesterday, so far, trying in and out all day... Nothing.. not a single catch... I hatched an egg though! LOL

**Comment by gsisso** [comment]
do the pokemons ran away or you dont see any pokemons?

**Comment by MDashK** [comment]
They don't even show up.
My ignore list is "16,19,21,41", but I will leave it empty and test it out.

**Comment by Edsardio** [comment]
You should double check if you have free pokemon slots... That was my problem

**Comment by MDashK** [comment]
Yes, I have free PKMN slots.
So, I'm running it now, and FINALLY, caught a Staryu.
Hmm... Maybe the ignore list is "ignoring" more PKMN than it should...

Maybe, if you ignore some PKMN; other won't show up, or something like that...

Just in under 7 min. it caught me 5, and none of them were in the ignore list.

**Comment by gsisso** [comment]
The ignore list is not based on pokemon id. #302

**Comment by MDashK** [comment]
Hmm... Maybe it was that that was making no PKMN at all to show up...
But... Since the values were wrong... Wasn't it supposed to give an error or simply ignore the values?

**Comment by SippieCup** [comment]
No. It will read that as a pokemon name. We can't do any verification
without violating Nintendo's IP.

**Comment by maxi2mc** [comment]
ignore list blocks everything :)

**Comment by MDashK** [comment]
It seems that way doesn't it? XD
I tried with names too, and nothing...

**Comment by maxi2mc** [comment]
I stopped using ignore list weeks ago, because I could not catch anything.

**Comment by SippieCup** [comment]
https://github.com/SwipeX/PokeMate/pull/225/files
